### DISCUSSION TITLE
Something between Lite and Pro

### DISCUSSION THREAD

**Discussion by martin-braun** [anonaddy/anonaddy#576] [Feature Requests]
I love to support the project. I'm on the lite plan. However, I want to add a 2nd domain, so that I can have international aliases and home country aliases side by side. Just upgrading to Pro to have a 2nd domain seems not worth it, because it costs 3 times as much. 

I would actually enjoy a tier in between, that has double the capacity for 24$/year. But, I would suggest to include 3 custom domains to have another backup domain, just in case.

It would be sad, if this would not be considered. After all, I could simply make another account on another lite plan, but I think this is against the ToS? Idk. Self-hosting is an option, but I have trust in you and would love to continue to fund the development of this great service.

Thank you!

**Comment by waldemarfm** [comment]
This would be easy money. I'm on the exact same situation. A "Standard" plan between Lite and Pro that offers 3 or 5 custom domains instead of 20 would be the perfect plan for my use case.

**Comment by MotafokaBR** [comment]
I have Addy Lite with a "developer" domain and it works great.
Now I got a second domain for a hobby, and my wife is cutting down on her self employed business, so no need for a dedicated webmail hosting anymore.

So where is the question again : Is there any plans for a subscription between Lite and Pro, for 2 or 3 custom domains?
